internal/driver/glfw/web_shortcuts_js.go

```diff
@@ -0,0 +1,19 @@
+//go:build js && !wasm
+// +build js,!wasm
+
+package glfw
+
+import (
+	"strings"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+// Checks if the browser is runnning on Mac OSX
+func isMacOSBrowser() bool {
+	if strings.Contains(strings.ToLower(js.Global.Get("window").Get("navigator").Get("platform").String()), "mac") {
+		return true
+	} else {
+		return false
+	}
```

Comment:
Just return the Boolean value directly instead of checking if it is true and then returning a true value.

Reply:
good point, I've made the changes